lab_2/main.py

```diff
@@ -12,8 +18,22 @@ def tokenize_by_lines(text: str) -> tuple:
     e.g. text = 'I have a cat.\nHis name is Bruno'
     --> (('i', 'have', 'a', 'cat'), ('his', 'name', 'is', 'bruno'))
     """
-    pass
+    from tokenizer import tokenize
 
+    if isinstance(text, str):
+        new_textt = []
+        #for i in text:
+        new_text = text.split('\n')
+        print(new_text)
+        for i in new_text:
+            token = tokenize(i)
+            tokens = new_textt.append(token)
```

Comment:
typo – `new_text`

Reply:
now you need to save these tokens somewhere to get a tuple of sentences with tokens